angular/core/components/invitation_form/invitation_form.coffee

```diff
@@ -6,7 +6,7 @@ angular.module('loomioApp').factory 'InvitationForm', ->
       _.filter Session.user().groups(), (group) ->
         AbilityService.canAddMembers(group)
 
-    $scope.form = Records.invitationForms.build(groupId: group.id or (_.first($scope.availableGroups()) or {}).id)
+    $scope.form = Records.invitationForms.build(groupId: Session.currentGroupId() or (_.first($scope.availableGroups()) or {}).id)
```

Comment:
Oh yeah. I was thinking that, instead of any of the forms knowing about Session.currentGroup, they're simply told which group they're rendering, so that we always have the option of rendering a form which isn't the current group.